cli/config/config_test.go

```diff
@@ -115,6 +115,7 @@ password`: "Invalid Auth config file",
 email`: "Invalid auth configuration file",
 	}
 
+	resetHomeDir()
```

Comment:
I'm a bit on the fence between calling this function for each test as first step, to make sure the path is clean for the test, OR defer it at first step, to make sure everything will be cleared for other tests. 🤔 
```go
func TestOldInvalidsAuth(t *testing.T) {
    defer resetHomeDir()
    ...
}
```
```go
func TestOldInvalidsAuth(t *testing.T) {
   resetHomeDir()
    ...
}
```

Reply:
It's tricky; any test that uses the `homeDir()` will initialise the package variable; my current approach was to put it in those tests that expect the variable to be re-resolved (because they patch the `HOME` variable for the test); changing the `HOME` variable is something that would only happen in tests (under normal circumstances it would not modify after the cli binary is started, so it would be safe to store it in the variable.